### ISSUE TITLE
allow and disallow vTypes on edges in marouter?

### ISSUE THREAD

**Issue by alexdawn** [eclipse/sumo#8237]
Is it possible for marouter to generate routes respecting the allowed and disallowed attributes of edges? Currently marouter is generate routes which duarouter is erroring on because of forbidden edges?

**Comment by behrisch** [comment]
No, this is currently not possible. I keep the issue open as a feature request.

**Comment by behrisch** [comment]
@bcoueraud87 for you
